datasources/instagram/search_instagram.py

```diff
@@ -10,6 +10,7 @@
 from backend.lib.search import Search
 from common.lib.item_mapping import MappedItem, MissingMappedField
 from common.lib.exceptions import WorkerInterruptedException, MapItemException
+from common.lib.helpers import UserInput
```

Comment:
Unnecessary import

Reply:
Removed in [c4b1943](https://github.com/digitalmethodsinitiative/4cat/pull/428/commits/c4b19434af52ecc2a57f7c8d609aa080ae5cdc15)